bundles/org.openhab.binding.ambientweather/src/main/java/org/openhab/binding/ambientweather/internal/processor/ProcessorInterface.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.ambientweather.internal.processor;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.binding.ambientweather.internal.handler.AmbientWeatherStationHandler;
+
+/**
+ * The {@link ProcessorInterface} is responsible for
+ *
+ * @author Mark Hilbush - Initial contribution
+ */
+@NonNullByDefault
+public interface ProcessorInterface {
+    /*
+     * Set the channel group Id for the station
+     */
+    public abstract void setChannelGroupId();
```

Comment:
`public abstract` is implicit for methods declared in interfaces, better leave it out.

Reply:
Done